modules/ipi-install-creating-a-disconnected-registry.adoc

```diff
@@ -5,254 +5,70 @@
 
 :_content-type: PROCEDURE
 [id="ipi-install-creating-a-disconnected-registry_{context}"]
-= (Optional) Creating a disconnected registry
+= Creating a disconnected registry on a registry node
```

Comment:
I'm not sure if we want to use `mirror registry` instead of `disconnected registry` in this section. 
Mirror registry is introduced in 
https://docs.openshift.com/container-platform/4.10/installing/disconnected_install/installing-mirroring-creating-registry.html and it sounds like more reasonable in the installation usage. `disconnected registry` sounds like the registry is located in a disconnected network, but it should have access to Quay at least. 
BTW, I see we're also saying mirror registry in the following steps.
```
This procedure explains how to install the mirror registry for Red Hat OpenShift on a local host
```

Reply:
@gpei are you referring to the terms (swapping the term 'mirror registry' for 'disconnected registry'), or are you talking about the tool ("Mirror Registry for Red Hat OpenShift") itself? Including the procedure for "Mirror Registry for Red Hat OpenShift" was encouraged by Eric Rich. 

Please note that I cannot swap 'mirror registry' for 'disconnected registry' if that's what you're suggesting. The terms "Mirror Registry for OpenShift" is for the actual tool itself; that can't be renamed because of Marketing. It's a confusing name with no flexibility. I tried to get it changed.